build.sbt

```diff
@@ -218,6 +218,8 @@ lazy val examples = project
   .settings(skip in publish := true)
   .settings(
     libraryDependencies ++= Seq(
+      "de.heikoseeberger"            %% "akka-http-circe"               % "1.31.0",
```

Comment:
Hmm... For someone not using Circe on their own and just wants caliban exposed with akka, this is easy to miss (that they need to add these 2 dependencies). It is quite different from the circe interop in core where you can only trigger the circe-related code if you use Circe.

I had another idea to support this problem of having too many combinations of backends and JSON libraries: integration with [tapir](https://github.com/softwaremill/tapir) or [endpoints](https://github.com/julienrf/endpoints). That would be another adapter that gives a tapir/endpoints route, then you can use it with the backend and json library of your choice. Not sure they support WebSocket though. The idea is similar to what I did with caliban-client where I just give an sttp request and users can use what they want. What do you think?